src/clr-angular/forms/tests/wrapper.spec.ts

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright (c) 2016-2018 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+import { TestBed } from '@angular/core/testing';
+import { FormsModule, NgControl } from '@angular/forms';
+import { By } from '@angular/platform-browser';
+
+import { ClrIconModule } from '../../icon/icon.module';
+import { ClrCommonFormsModule } from '../common/common.module';
+import { IfErrorService } from '../common/if-error/if-error.service';
+
+import { NgControlService } from '../common/providers/ng-control.service';
+import { Layouts, LayoutService } from '../common/providers/layout.service';
+
+export function WrapperNoLabelSpec(testContainer, testControl, testComponent): void {
+  describe('no label', () => {
+    let fixture, containerDE, containerEl, layoutService;
+    beforeEach(() => {
+      TestBed.configureTestingModule({
+        imports: [ClrIconModule, ClrCommonFormsModule, FormsModule],
+        declarations: [testContainer, testControl, testComponent],
+        providers: [NgControl, NgControlService, IfErrorService, LayoutService],
+      });
+      fixture = TestBed.createComponent(testComponent);
+
+      containerDE = fixture.debugElement.query(By.directive(testContainer));
+      containerEl = containerDE.nativeElement;
+      layoutService = containerDE.injector.get(LayoutService);
+    });
+
+    it('adds an empty label when instantiated without vertical layout', () => {
```

Comment:
I don't understand this test. The empty label for the wrapper (not the container) has nothing to do with layout, it's necessary just to display a radio button. The implementation in the wrapper is correct, but this test has nothing to do with what we actually are testing.

Reply:
This test should just check the host wrapping is wired up correctly in this case, since its a pure use of the host wrapper. I removed the layout part.